bin/middlewares/og_meta.js

```diff
@@ -0,0 +1,91 @@
+const request = require('request')
+const acceptLanguage = require('accept-language')
+const QueryError = require('../query_error')
+
+const metas = [
+  {name : 'site_name', content : 'Qwant map'},
+  {name : 'image', content : '/images/maps_opengraph.png'},
+  {name : 'url', content : 'www.qwant.com/maps'},
```

Comment:
Could the url be set dynamically based on the request ?

Reply:
Je veux pas le faire car non trivial 
l'url se genere avec `${req.protocol}://${req.get('host')}` mais alors il manque la query (/maps)
mais si j'ajoute queryUrl alors on aura aussi /place à la suite

Reply:
I am not sure. We also need to set a canonical url when the page is about a specific poi.